tests/logictest/suites/base/03_dml/03_0025_delete_from

```diff
@@ -140,6 +140,29 @@ select count(*) = 0 from t;
 
 
 
+statement ok
```

Comment:
How about adding a stateful test based on `ontime` and `hits` in which we will delete data on different condition and check the remaining count.

Reply:
`ontime/hits` are stateful tests, the delete should be a stateless test first, but I think we need more test cases here, such as generating a table from `select ... from numbers(100)` and doing some delete operations on this table.

Reply:
Added test in [03_0025_delete_from](https://github.com/datafuselabs/databend/pull/8824/files/7700c95cbb534196d9743b80b314653afc4a682d#diff-e06a0a882879de1a4f36a1f8e8e949f101e8a71fc6dc3eab407b256e80682704). We had a number of test cases  to cover various scenarios previously.